src/main/kotlin/org/jitsi/nlj/util/StreamInformation.kt

```diff
@@ -47,6 +55,12 @@ class StreamInformationStoreImpl : StreamInformationStore {
     override val rtpExtensions: List<RtpExtension>
         get() = _rtpExtensions
 
+    private val payloadTypesLock = Any()
+    private val payloadTypeHandlers = mutableListOf<RtpPayloadTypesChangedHandler>()
+    private val _rtpPayloadTypes = mutableMapOf<Byte, PayloadType>()
```

Comment:
Should this be a ConcurrentHashMap, since the read-only access to this does not use the lock?